
20230605 16:55:40 INFO: 2023-06-05 16:21:48882spelling_with_letters_smart.notebook is 'Review' Task
NoneType: None

20230605 16:55:41 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:41 INFO: 2023-06-05 16:21:44876Perimeter.notebook is 'Review' Task
NoneType: None

20230605 16:55:41 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:41 INFO: 2023-06-05 16:21:42874If_I_writing_prompts_smart.notebook is 'Review' Task
NoneType: None

20230605 16:55:42 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:42 INFO: 2023-06-05 16:21:40862ordering_numbers_smart.notebook is 'Review' Task
NoneType: None

20230605 16:55:42 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:42 INFO: 2023-06-05 16:21:34859Monkey_Themed_Math_Test_Prep.notebook is 'Review' Task
NoneType: None

20230605 16:55:42 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:42 INFO: 2023-06-05 16:21:29855math_word_problems_smart.notebook is 'Review' Task
NoneType: None

20230605 16:55:43 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:43 INFO: 2023-06-05 16:21:26853greater_than_less_than_smart.notebook is 'Review' Task
NoneType: None

20230605 16:55:43 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:43 INFO: 2023-06-05 16:21:21847multiplication_facts_race.notebook is 'Review' Task
NoneType: None

20230605 16:55:43 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:44 INFO: 2023-06-05 16:21:17847Halloween_Squares.notebook is 'Review' Task
NoneType: None

20230605 16:55:44 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:55:44 INFO: 2023-06-05 16:20:01501student_picker_watermelon.notebook is 'Review' Task
NoneType: None

20230605 16:55:44 INFO: 找不到vBoard.exe
Traceback (most recent call last):
  File "C:\FileConverter\test.py", line 71, in test_
    self.test.start_web_download_flow(i)
  File "C:\FileConverter\test.py", line 13, in start_web_download_flow
    self.wf.download_file(taskIndex= taskIndex)
  File "C:\FileConverter\Flow\Conversion_Flow.py", line 57, in download_file
    raise
RuntimeError: No active exception to reraise

20230605 16:56:01 INFO: C:\Program Files\ViewSonic\vBoard\vBoard.exe已成功開啟.
NoneType: None
